nomad.yaml

```diff
@@ -6,7 +6,7 @@ plugins:
   # We only include our schema here. Without the explicit include, all plugins will be
   # loaded. Many build in plugins require more dependencies. Install nomad-lab[parsing]
   # to make all default plugins work.
-  include: 'schema/nomad_simulations'
+  include: 'schemas/nomad_simulations'
   options:
     schema/nomad_simulations:
```

Comment:
here too, `schemas/nomad_simulations`

Reply:
Isn't it already plural?

Reply:
no

Reply:
You have to change the options to be plural as well, i.e., `schemas/nomad_simulations:`